openslides_backend/migrations/migrations/0038_remove_user_default_projector.py

```diff
@@ -0,0 +1,60 @@
+from typing import Any, Dict, List, Optional
+
+from datastore.migrations.core.base_migration import BaseMigration
+from datastore.migrations.core.events import (
+    BaseEvent,
+    CreateEvent,
+    DeleteFieldsEvent,
+    ListUpdateEvent,
+    UpdateEvent,
+)
+from datastore.shared.util import collection_from_fqid
+
+
+class Migration(BaseMigration):
+    """
+    This migration removes the field `meeting/default_projector_$user_ids`.
+    """
+
+    target_migration_index = 39
+    collection_fields_map = {
+        "meeting": ["default_projector_$user_ids"],
+        "projector": ["used_as_default_$user_in_meeting_id"],
+    }
+
+    def remove_fields(self, obj: Dict[str, Any], fields: List[str]) -> None:
+        for field in fields:
+            if field in obj:
+                del obj[field]
+
+    def remove_replacement(self, obj: Dict[str, Any]) -> None:
+        for field in ("default_projector_$_ids", "used_as_default_$_in_meeting_id"):
```

Comment:
You can pass the collection to this method adn then iterate over `collection_fields_map[collection] instead. Removes unnecessary iterations and duplication fo the field names.

Reply:
Nope, the fields in collection_fields_map are different.

Reply:
I used a collection_replacement_fields_map for the other fields.

Reply:
Ah right, that's fine then.